crowdin.yml

```diff
@@ -1,8 +1,8 @@
 files:
-  - source: /**/*.md
+  - source: /**/@(*.md|sidebar_translations.json)
```

Comment:
I guess this works, it might have been cleaner to just have a seperate `-source` for the sidebar json?